### DISCUSSION TITLE
question: adding ot-table to the font using `ttx -m`

### DISCUSSION THREAD

**Discussion by aminabedi68** [fonttools/fonttools#2560] [Q&A] [answered]
Hi
(i don't know where is the right place to this question, if you think it's not belong here please let me know)
I'm trying to fix google font bakery report issues. it's all done except `prep` table. i have a primary version of this table in ttx-XML and i hope ttx could add it to the font during build process(i don't want to use gf-tools). i tried that in many ways like:
 `ttx -m fontname.ttf table.ttx`
, but didn't work to me, so how can i do that?

**Comment by khaledhosny** [comment]
I think the order of arguments is reversed.

**Comment by aminabedi68** [comment, marked as answer]
hi @khaledhosny 
no, still not working.

**Reply by khaledhosny** [reply]
So your the original order was right, and I tried it and it does work. What error do you get or what output are you getting? How did you generate the `table.ttx` file? Mine looks like this:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<ttFont sfntVersion="\x00\x01\x00\x00" ttLibVersion="4.31">

  <prep>
    <assembly>
      PUSHW[ ]  /* 1 value pushed */
      511
      SCANCTRL[ ]       /* ScanConversionControl */
      PUSHB[ ]  /* 1 value pushed */
      4
      SCANTYPE[ ]       /* ScanType */
    </assembly>
  </prep>

</ttFont>
```

**Reply by aminabedi68** [reply]
it's working now! i forgot `<ttFont></ttFont>`. thank you :)
